src/backend/vulkan/src/device.rs

```diff
@@ -1483,6 +1484,13 @@ impl d::Device<B> for Device {
         (swapchain, Backbuffer::Images(images))
     }
 
+    fn destroy_swapchain(&self, swapchain: w::Swapchain) -> () {
```

Comment:
I *hope* this is right, but I'm not sure - hard to test things for backends that you don't have :/ https://github.com/MaikKlein/ash/blob/master/ash/src/extensions/swapchain.rs#L37